lib/compress.js

```diff
@@ -77,6 +77,7 @@ function Compressor(options, false_by_default) {
         screw_ie8     : true,
         sequences     : !false_by_default,
         side_effects  : !false_by_default,
+        switches      : !false_by_default,
```

Comment:
I did start with `switch`, but then reading your examples again and was forced to pick consistency over language.

If we want to go all out evil we can change them all into singular form in 3.x